applications/FluidDynamicsApplication/python_scripts/apply_inlet_process.py

```diff
@@ -8,22 +8,20 @@ def Factory(settings, Model):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
     return ApplyInletProcess(Model, settings["Parameters"])
 
+def GetHistoricalVariables(settings):
+    copy_settings = settings.__deepcopy__()
```

Comment:
```suggestion
    copy_settings = settings.Clone()
```

Should also be possible.